driver-sync/src/test/functional/com/mongodb/client/AbstractClientSideOperationsTimeoutProseTest.java

```diff
@@ -700,6 +700,14 @@ public void test10CustomTestWithTransactionUsesASingleTimeoutWithLock() {
         }
     }
 
+    @DisplayName("11. Multi-batch bulkWrites")
+    @Test
+    void test11MultiBatchBulkWrites() {
+        assumeTrue(serverVersionAtLeast(8, 0));
+        assumeFalse(isServerlessTest());
+        assumeTrue(Runtime.getRuntime().availableProcessors() < 1, "BULK-TODO implement prose test https://github.com/mongodb/specifications/blob/master/source/client-side-operations-timeout/tests/README.md#11-multi-batch-bulkwrites");
```

Comment:
Does this have a jira ticket - if not lets add one and then that can be tracked there.